frontend/src/features/qa/page-list/helpers/severity.ts

```diff
@@ -22,3 +23,16 @@ export const severityFromResourceCounts = cached(
     return "good";
   },
 );
+
+export const textColorFromSeverity = cached((severity: Severity) => {
+  switch (severity) {
+    case "good":
+      return tw`text-green-600`;
```

Comment:
Ah I haven't thought of using `tw` in utilities in this way, cool!

Reply:
Yeah, it's handy! In this case we don't _actually_ really need it, cause we're definitely using all these classes elsewhere already, but if we weren't this would ensure that the compiler knows to include them.